lib/structures/Invite.js

```diff
@@ -33,12 +37,15 @@ class Invite extends Base {
         this.channel = data.channel;
         if(data.guild) {
             this.guild = {
-                splash: data.guild.splash,
-                icon: data.guild.icon,
                 id: data.guild.id,
                 name: data.guild.name,
-                textChannelCount: data.guild.text_channel_count !== undefined ? data.guild.text_channel_count : null,
-                voiceChannelCount: data.guild.voice_channel_count !== undefined ? data.guild.voice_channel_count : null
+                splash: data.guild.splash,
```

Comment:
What happens if `guild` is undefined? 
![image](https://user-images.githubusercontent.com/23035000/71998699-1c109080-320e-11ea-9f60-cbea9b420ec1.png)

Reply:
![image](https://user-images.githubusercontent.com/17226394/72008030-9514e580-3218-11ea-8ce6-b534d20ac6bc.png)